src/utils/renderBatch.js

```diff
@@ -17,7 +17,7 @@ export default (config, isClosing) => (req, res) => {
       if (isClosing()) {
         logger.info('Ending request when closing!');
       }
-      res.status(manager.statusCode).json(manager.getResults()).end();
+      return res.status(manager.statusCode).json(manager.getResults()).end();
```

Comment:
this change however doesn't do much, I believe end() returns undefined so perhaps it's better to explicitly return undefined or null